powerquery-docs/connectors/celonis-ems.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Power Query Celonis EMS connector
+description: Provides basic information, prerequisites, and connection steps for Celonis EMS
+author: capono
+
+ms.date: 02/17/2023 
+ms.author: 
+LocalizationGroup: reference
+---
+
+# Celonis EMS
+
+  
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI (Datasets) |
+| Authentication Types Supported | Organizational Account |
+| Function Reference Documentation | [Web.Contents](https://learn.microsoft.com/en-us/powerquery-m/web-contents) </br> [Parquet.Document](xxx) </br> [Json.Document](https://learn.microsoft.com/en-us/powerquery-m/json-document) </br> [Json.FromValue](https://learn.microsoft.com/en-us/powerquery-m/json-fromvalue)|
+
+  
+
+## Prerequisites
+
+Before you can sign in to [Celonis EMS](https://www.celonis.com/ems/platform/?utm_source=google&utm_medium=cpc&utm_campaign=evergreen&utm_content=en_maxconvvalueems_platform_eta&creative=605673291817&keyword=celonis%20ems&matchtype=e&network=g&device=c&_bt=605673291817&_bk=celonis%20ems&_bm=e&_bn=g&_bg=134342462066&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhD5aEPA6iavBUsBG1p0WeGEVi9OF5qcOaohHqNgpbuWa-f9ZHwCtHhoCSqYQAvD_BwE), you must have a Celonis EMS account (username/password).
```

Comment:
This URL has a lot of parameters... can we simply it?

Reply:
Was simplified.